test/support/mix_generator_helper.exs

```diff
@@ -0,0 +1,77 @@
+defmodule Kitto.MixGeneratorHelper do
+  @moduledoc """
+  Helpers to use when testing Mix tasks
+  """
+  import ExUnit.Assertions
+  import Mock
+  alias Kitto.Mix.GeneratorMock
+
+  @mocks [create_file: &GeneratorMock.create_file/2,
+          create_directory: &GeneratorMock.create_directory/1]
+
+  @doc """
+  Asserts that a file was created (with Mix.Generator.create_file/2) by running
+  a Mix task.
+
+  Arguments:
+
+    * `file`: The file expected to be created. Can be a regex or string
+    * `task`: Function which runs the task that is expected to create a file
+
+  Usage:
+
+      assert_creates_file ~r/my_job.ex/, fn() ->
+        Mix.Kitto.Tasks.Gen.Job.run(["my_job"])
+      end
+  """
+  def assert_creates_file(file, task) do
```

Comment:
Can't we just use https://github.com/kittoframework/kitto/blob/master/test/support/file_assertion_helper.exs#L9 here?